cypress/integration/pages/frontPage/testsForCanonicalOnly.js

```diff
@@ -11,6 +11,28 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({ service }) => {
   if (Cypress.env('APP_ENV') === 'local') {
     runCanonicalAdsTests({ service });
   }
+  describe(`No testsToAlwaysRunForCanonicalOnly to run for ${service}`, () => {
+    describe(`US Election Banner`, () => {
+      it('should have US Election banner for Hindi, Arabic, Portuguese, Mundo, Persian, Russian', () => {
+        const servicesWithBanner = [
+          'hindi',
+          'arabic',
+          'portuguese',
+          'mundo',
+          'persian',
+          'russian',
+        ];
+        if (servicesWithBanner.includes(service)) {
+          cy.get('[class^="gel-wrap"]').should('be.visible');
+          cy.get('[class^="IndexPageContainer-sc-1yb6vt4-0"]').should(
```

Comment:
I am wondering if we could add a data-e2e attribute to the banner wrapper, so that we can check for its existence, rather than relying on class names (which could change).

Reply:
https://github.com/bbc/simorgh/blob/latest/src/app/containers/USElectionBanner/index.jsx#L47

You could update this to be:
```
<FrontPageSection data-e2e='us-election-banner'>
```

Reply:
Is FrontPageSection only ever there with the banner, or is it used for other things too and could be present without the banner?

Reply:
FrontPageSections are used for other things, without the banner, but if we only add the `data-e2e` attribute to it in the ElectionBanner container, nothing else will be affected, if that makes sense?